### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Bharathwaj-Murali**
Made changes to Configure Incident Mirroring section to account for the new added feature of Use Display Value

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by yaakovpraisler** [comment]
Hi @Bharathwaj-Murali, Thank you for the contribution!
@Shellyber - Can you please have a look here?

File: Packs/ServiceNow/Integrations/ServiceNowv2/README.md [hunk 1]
```diff
@@ -144,15 +144,13 @@ To set up incident mirroring you need to:
 To use ServiceNow on Cortex XSOAR, ensure your service account has the following roles required to make API calls:  
 - Rest_api_explorer
 - Snc_platform_rest_api_access
-- Itil (Needed to access the sys_journal_field to allow comments, work notes, and files to be mirrored in and out. Incoming mirroring queries sys_journal_field for comments and work notes and therefore, access to sys_journal_table is also required.)  
-**Note:**  
-If your organization does not allow assigning the Itil role, you need to give service account elevated
-privileges to the sys_journal_field (see this ServiceNow community link for [giving elevated read access](https://community.servicenow.com/community?id=community_question&sys_id=b4051bf4db4c1cd823f4a345ca9619dc) and potential risks).
-- Read access to sys_journal_field is required for incoming mirroring.
+- itil or read/write access the specific tables you want to have access to. If you choose to give permissions only for specific tables, make sure you have the correct role so you have permissions to work with the relevant table. Keep in mind that these permissions may not suffice for managing records in some tables.
```
Comment (Shellyber) [inline Packs/ServiceNow/Integrations/ServiceNowv2/README.md:147]: Why did you change that?
This permission is basically for mirroring comments. What other use cases/tables can be?

I'm not sure we need to change this part

File: Packs/ServiceNow/Integrations/ServiceNowv2/README.md [hunk 2]
```diff
@@ -180,19 +178,24 @@ custom mapping, follow the instructions in STEP 3 and then select the custom map
     - **In** - Mirrors changes on the ServiceNow ticket in to the Cortex XSOAR ticket.
     - **Out** - Mirrors changes on the Cortex XSOAR ticket to the ServiceNow ticket.
     - **Both** - Mirrors changes both in and out on both tickets.
-11. Set the Timestamp field to query as part of the mirroring flow. This defines the ticket_last_update - the epoch timestamp when the ServiceNow incident was last updated. The default is sys_updated_on.
-12. Enter the relevant **Comment Entry Tag**, **Work Note Entry Tag**, **File Entry Tag To ServiceNow** and **File Entry Tag From ServiceNow** values.  
+11. Enable the checkbox for **Use Display Value** if you want to fetch comments and work notes without using sys_journal_field table which required an elevated read only permission.
```
Comment (Shellyber) [inline Packs/ServiceNow/Integrations/ServiceNowv2/README.md:181]: Nice!

**Review by Shellyber — COMMENTED** [review]
Hi, @Bharathwaj-Murali Thank you for your contribution, that looks great! 
Please see my first note, I'm not sure regarding other use cases there if you can explain.

File: Packs/ServiceNow/Integrations/ServiceNowv2/README.md (hunk 1 above)
Comment (Bharathwaj-Murali) [reply Packs/ServiceNow/Integrations/ServiceNowv2/README.md:147]: @Shellyber  The new FR that was rolled out which uses 'Use Display Value' makes this as optional. Clients can now bypass the elevated privileges needed in order for comments and work notes to be accessed or mirrored. Hence this means that ITIL is no longer an essential privilege. They can either use ITIL or just ensure we have sufficient access to get all the records in the table. This is why I edited the notes to make it sequential and a little bit clearer to make sure clients understand their options.  Also noting that, if mirroring of comments and work notes fail, it would break the entire mirror cycle and nothing would be mirrored in. which makes this an important step to be clarified.

Comment (Shellyber) [reply Packs/ServiceNow/Integrations/ServiceNowv2/README.md:147]: @Bharathwaj-Murali I would probably write then that the Itil is not mandatory anymore since we have the new feature. 
But I don't think we need to change and add the rest.
I'll leave the third point as is and add a note that this is no longer required for mirroring comments and work notes and refer to the new functionality

Comment (Bharathwaj-Murali) [reply Packs/ServiceNow/Integrations/ServiceNowv2/README.md:147]: @Shellyber I did not change it. I just combined the footnote to the itil permission as they are related. And I moved the sys_journal_field to a separate point. I will reformat it.

**Review by Shellyber — APPROVED** [review]
Nice Work
